pages/support.tsx

```diff
@@ -0,0 +1,241 @@
+import React from 'react';
+import Head from 'next/head';
+import Footer from 'src/views/home/Footer';
+import Image from 'next/image';
+
+type ChildProp = {
+  children: React.ReactElement | Array<React.ReactElement | string> | string;
+  className?: string;
+}
+
+const Heading = function ({ children, className }: ChildProp): JSX.Element {
```

Comment:
Not 100% necessary, but we're moving to using this pattern for declaring functional components:

```
function Heading(props: ChildProp): JSX.Element {...}
```

Reply:
They have been updated to the new style